helm/values.yaml

```diff
@@ -47,56 +55,43 @@ validators:
   #   password: notarypass
   #   # or
   #   secret_name: notarysecret  # define a k8s secret with `username` and `password` fields
-- name: docker_essentials  # public notary form dockerhub
+# pre-configured nv1 validator for public notary from Docker Hub
+- name: dockerhub_essentials
   type: notaryv1
   host: notary.docker.io
   pub_keys:
-  # this is the public key for all official dockerhub images (e.g. redis)
+    # public key for official docker images (https://hub.docker.com/search?q=&type=image&image_filter=official)
   - name: docker_official
     key: |
       -----BEGIN PUBLIC KEY-----
       MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEOXYta5TgdCwXTCnLU09W5T4M4r9f
       QQrqJuADP6U7g5r9ICgPSmZuRHP/1AYUfOQW3baveKsT969EfELKj1lfCA==
       -----END PUBLIC KEY-----
-  # this is the public key to the `securesystemsengineering` repo on dockerhub
+  # public key for Connaisseur images
   - name: sse_official
     key: |
       -----BEGIN PUBLIC KEY-----
       MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEsx28WV7BsQfnHF1kZmpdCTTLJaWe
       d0CA+JOi8H4REuBaWSZ5zPDe468WuOJ6f71E7WFg3CVEVYHuoZt2UYbN/Q==
       -----END PUBLIC KEY-----
-- name: cosign_demo  # for verifying cosign signatures
-  type: cosign
-  pub_keys:
-  - name: sse
-    key: |
-      -----BEGIN PUBLIC KEY-----
-      MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni
-      qiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==
-      -----END PUBLIC KEY-----
 
-# the image policy, which defines all repositories that need to be
-# verified. more detail in the git repo README.md
+### IMAGE POLICY ###
+# the image policy ties validators and images together whereby always only the most specific rule (pattern)
+# is applied. Specify if and how images should be validated by which validator.
 policy:
 - pattern: "*:*"
-  validator: dockerhub
 - pattern: "docker.io/library/*:*"
-  validator: dockerhub
+  validator: dockerhub_essentials
   with:
     key: docker_official
 - pattern: "k8s.gcr.io/*:*"
   validator: allow
-- pattern: "docker.io/securesystemsengineering/connaisseur:*"
-  validator: dockerhub
+- pattern: "docker.io/securesystemsengineering/*:*"
+  validator: dockerhub_essentials
   with:
-    key: sse
+    key: sse_official
 - pattern: "docker.io/securesystemsengineering/connaisseur:helm-hook-*"
   validator: allow
```

Comment:
That this stays at allow is the chicken-egg problem during bootstrapping?

Reply:
fixed it